test/server/lib/DocApi.ts

```diff
@@ -2530,6 +2530,27 @@ function testDocApi(settings: {
       ]);
     });
 
+    it("GET /docs/{did}/attachments/download downloads all attachments as a .tar", async function () {
```

Comment:
I noticed on previous work that @berhalak later needed to add forwarders for new attachment endpoints:
https://github.com/gristlabs/grist-core/pull/1348/files#diff-531fd1025fc38b2b06fc3bafb8972112a3480fa807426e12e7b047374576c807

This is normal, in multi-server Grist all doc workers interact with the outside world via forwarding from home servers for ... reasons.

I was wondering why the tests you're adding are passing, despite being run for multiple configurations, including separate home server and doc worker.

I think it may be that by using serverUrl versus homeUrl, the tests are talking directly to the doc worker.

Reply:
I've added this -  but not quite sure how to test it!

Reply:
I think the trick will be to use `homeUrl` rather than `serverUrl` to better mimic real conditions. But I could be misremembering.